### PR TITLE
Users logic

### PR DESCRIPTION
**Pull request by KatrinaSidorenko**

### CODE REVIEW

File: api/ChattO/API/DTOs/Requests/User/UpdateUserRequest.cs [hunk 1]
```diff
@@ -1,16 +1,29 @@
...
+public class UpdateUserRequest : IMapWith<UpdateAppUser.Command>
 {
     public string? DisplayName { get; set; }
 
     public string? Email { get; set; }
 
     public string? UserName { get; set; }
 
-    public IFormFile? ProfileImage { get; set; } 
+    //public IFormFile? ProfileImage { get; set; }
```
Comment (NureBieloshapkaMykyta) [inline api/ChattO/API/DTOs/Requests/User/UpdateUserRequest.cs:17]: why commented

File: api/ChattO/Infrastructure/Services/DataAccess/Repository.cs [hunk 2]
```diff
@@ -174,6 +174,38 @@ public async Task<Result<int>> GetTotalCountAsync(Expression<Func<TEntity, bool>
         }
     }
 
+    public async Task<Result<bool>> PartialUpdateAsync(Guid id, object entity)
```
Comment (NureBieloshapkaMykyta) [inline api/ChattO/Infrastructure/Services/DataAccess/Repository.cs]: I think we can make it generic

File: api/ChattO/API/Controllers/UserController.cs [hunk 3]
```diff
@@ -37,60 +38,37 @@ public async Task<IActionResult> Create([FromBody] CreateAccountsRequest request
...
     public async Task<IActionResult> GetById([FromRoute] Guid id)
     {
-        return Ok();
+        var userResult = await Mediator.Send(new GetDetailsAppUser.Query { Id = id });
+        if (!userResult.IsSuccessful)
+            return HandleResult(userResult);
+
+        var response = Mapper.Map<UserDetailsResponse>(userResult.Data);
+        response.Organization = Mapper.Map<GetDetailsOrganizationResponse>(userResult.Data.Organization);
+        response.ProfileImage = Mapper.Map<FileResponse>(userResult.Data.ProfileImage);
```
Comment (NureBieloshapkaMykyta) [inline api/ChattO/API/Controllers/UserController.cs]: I believe it will work in single map operation "Mapper.Map<UserDetailsResponse>(userResult.Data)"

**Review by NureBieloshapkaMykyta — CHANGES_REQUESTED** [review]

File: api/ChattO/API/Controllers/UserController.cs (hunk 3 above)
Comment (KatrinaSidorenko) [reply api/ChattO/API/Controllers/UserController.cs]: Fixed

File: api/ChattO/Infrastructure/Services/DataAccess/Repository.cs (hunk 2 above)
Comment (KatrinaSidorenko) [reply api/ChattO/Infrastructure/Services/DataAccess/Repository.cs]: Fixed

**Review by NureBieloshapkaMykyta — APPROVED** [review]

File: api/ChattO/API/DTOs/Requests/User/UpdateUserRequest.cs (hunk 1 above)
Comment (KatrinaSidorenko) [reply api/ChattO/API/DTOs/Requests/User/UpdateUserRequest.cs:17]: We can't updatethe picture through the PATCH method.

**Pull request merged**